AsyncDisplayKit/ASCollectionView.h

```diff
@@ -420,6 +423,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)collectionViewUnlockDataSource:(ASCollectionView *)collectionView ASDISPLAYNODE_DEPRECATED;
 
+- (_Nonnull id<ASSectionContext>)collectionView:(ASCollectionView *)collectionView contextForSectionAtIndex:(NSInteger)sectionIndex;
```

Comment:
Actually if we create an object for each section that holds a nullable context, we can relax on the nullability of this method. Will work on it.